client/js/shout.js

```diff
@@ -191,13 +197,7 @@ $(function() {
 		var chan = chat.find(target);
 		var from = data.msg.from;
 
-		var msg = $(render("msg", {messages: [data.msg]}));
```

Comment:
I guess removing `{messages: ...}` echoes to [this change](https://github.com/erming/shout/pull/596/files?diff=split#diff-a65d27a1899225d694874754184eea4bL29). Just out of curiosity, why removing this?